Habitica/src/main/java/com/habitrpg/android/habitica/ui/fragments/preferences/PreferencesFragment.kt

```diff
@@ -209,7 +214,11 @@ class PreferencesFragment : BasePreferencesFragment(), SharedPreferences.OnShare
                 pushNotificationsPreference?.isEnabled = usePushNotifications
                 userRepository.updateUser("preferences.pushNotifications.unsubscribeFromAll", !usePushNotifications).subscribe()
                 if (usePushNotifications) {
-                    pushNotificationManager.addPushDeviceUsingStoredToken()
+                    if (!pushNotificationManager.notificationPermissionEnabled() && (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU)) {

```

Comment:
Is it actually necessary to check for tiramisu? Since that is the only version where the user would even need to approve notifications, the `notificationPermissionEnabled` call should only return `false` on devices with android 13 or higher. Right?